projects/plugins/jetpack/modules/comments/comments.php

```diff
@@ -536,6 +536,11 @@ public function watch_comment_parent() {
 	public function pre_comment_on_post() {
 		$post_array = stripslashes_deep( $_POST );
 
+		/** This filter is documented in modules/comments/comments.php */
+		if ( ! apply_filters( 'jetpack_comment_form_enabled_for_' . get_post_type( $post_array['comment_post_ID'] ), true ) ) {
```

Comment:
This won't work as-is. Realized there's this same check at the bottom of this method that's die'ing out intentionally. We'll need another solution.

Reply:
Worked on a separate solution over here -  #23945 - would that do that job?

Reply:
Great! Thanks for taking it 👍